<% if object.errors.any? %>
   <div id="error_explanation">
     <h2><%= pluralize(object.errors.count, "erreur")%> ont emp&egrave;ch&eacute; la sauvegarde:</h2>
     <p>Il y avait les probl&egrave;mes suivants:</p>
     <ul>
     <% object.errors.full_messages.each do |msg| %>
       <li><%= msg %></li>
     <% end %>
     </ul>
   </div>
<% end %>
